js/index.js

```diff
@@ -8,35 +8,121 @@ const siteContent = {
+    "services-h4": "Services",
+    "services-content":
+      "Services content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.",
+    "product-h4": "Product",
+    "product-content":
+      "Product content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.",
+    "vision-h4": "Vision",
+    "vision-content":
+      "Vision content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis."
   },
-  "contact": {
-    "contact-h4" : "Contact",
-    "address" : "123 Way 456 Street Somewhere, USA",
-    "phone" : "1 (888) 888-8888",
-    "email" : "sales@greatidea.io",
-  },
-  "footer": {
-    "copyright" : "Copyright Great Idea! 2018"
+  contact: {
+    "contact-h4": "Contact",
+    address: "123 Way 456 Street Somewhere, USA",
+    phone: "1 (888) 888-8888",
+    email: "sales@greatidea.io"
   },
+  footer: {
+    copyright: "Copyright Great Idea! 2018"
+  }
 };
 
 // Example: Update the img src for the logo
+//let logo = document.getElementById("logo-img");
+//logo.setAttribute("src", siteContent["nav"]["img-src"]);
+
+// nav selectors
+const [nav] = document.getElementsByTagName("nav");
 let logo = document.getElementById("logo-img");
-logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+// cta selectors
+const [cta] = document.getElementsByClassName("cta");
+const [ctaH1] = cta.getElementsByTagName("h1");
+const [ctaBtn] = cta.getElementsByTagName("button");
+const ctaImg = document.getElementById("cta-img");
+
+//main content selectors
+const [maincontent] = document.getElementsByClassName("main-content");
+const myh4 = maincontent.getElementsByTagName("h4");
+const myp = maincontent.querySelectorAll("p");
+const mainImg = document.getElementById("middle-img");
+
+//contact section selectors
+const [contact] = document.getElementsByClassName("contact");
+const mycontacth4 = contact.getElementsByTagName("h4");
+const mycontactp = contact.querySelectorAll("p");
+
+//footer section selectors
+const [footer] = document.getElementsByTagName("footer");
+
+// update nav links
+Array.from(nav.children).forEach((val, idx) => {
+  val.textContent = siteContent["nav"][`nav-item-${idx + 1}`];
+});
+
+//make navigation green
+const navchange = document.querySelectorAll("a");
+
+navchange[0].style.color = "green";
+navchange[1].style.color = "green";
+navchange[2].style.color = "green";
+navchange[3].style.color = "green";
+navchange[4].style.color = "green";
+navchange[5].style.color = "green";
```

Comment:
Oooh, we're repeating code here. What if we had a hundred nav elements? Or a thousand? That'd be a lot of assignments to type out. So how could we make it more DRY?

Well, I see an opportunity here for loops! We just need to make sure that we're using the correct array of values.
```js
for (const link of navchange) {
  link.style.color = 'green';
}
```